Float.FileDownloader.Tests/DownloadRequest.Tests.cs

```diff
@@ -72,5 +74,18 @@ public async Task TestServerNotFound()
             var request = new HttpRequestMessage(HttpMethod.Get, "http://www.floatfloatfloat.float/float");
             await Assert.ThrowsAsync<HttpRequestException>(async () => await DownloadRequest.Download(request, TempFilePath()));
         }
+
+        [Fact]
+        public async Task TestNoCookies()
```

Comment:
@EBusch Love that we're updating tests here -- can you demonstrate the updated tests failing before the fix in `DownloadRequest`?